components/opponent/wikis/warcraft/opponent_display_custom.lua

```diff
@@ -44,7 +44,7 @@ function CustomOpponentDisplay.InlineOpponent(props)
 		return CustomOpponentDisplay.InlinePlayers(props)
 	end
 
-	return OpponentDisplay.InlineOpponent(props)
+	return OpponentDisplay.InlineOpponent(props --[[@as InlineOpponentProps]])
```

Comment:
Couldn't you make `---@class WarcraftInlineOpponentProps : InlineOpponentProps` instead?

Reply:
yeah, better